kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/spec/Spec.kt

```diff
@@ -305,6 +306,13 @@ abstract class Spec : TestConfiguration() {
 
    var coroutineTestScope: Boolean? = null
 
+   /**
+    * Flag controlling whether virtual time is enabled for non-deterministic test functions (`eventually`, ...).
+    * This is for Kotest-internal testing only and should not be used outside Kotest.
+    */
+   @KotestInternal
+   var nonDeterministicTestVirtualTimeEnabled = false
+
```

Comment:
👍

Reply:
```suggestion
   /**
    * Flag controlling whether virtual time is enabled for non-deterministic test functions (`eventually`, ...).
    * This is for Kotest-internal testing only and should not be used outside Kotest.
    */
   @KotestInternal
   var nonDeterministicTestVirtualTimeEnabled: Boolean = false

```

A minor nitpick of mine: I think it's best to explicitly state the type, especially in non-private functions. imo it's easier to scan and to read, and indicates a stricter contract. WDYT?